### DISCUSSION TITLE
Supplemental Guidance Template Discussion

### DISCUSSION THREAD

**Discussion by mraccess77** [w3c/low-vision-a11y-tf#137] [Ideas]
This discussion is created as a place where we can discuss the supplemental guidance template sections and content.  Please provide discussion comments below on the categories, content, order, etc. for the templates. 

Keep in mind that we don't have to have this finalized to create supplemental guidance and this template is still evolving.

- User [ Stories | Needs | Experience | Issues | Objectives ]
- What To Do
- How it Helps
- [ More Details | Description ]
- [ Getting Started ]
- Examples
- Support
- Related WCAG
- Related Resources
- Key Terms
- Editorial Notes
### User [ Stories | Needs | Experience | Issues | Objectives ]
{ see examples in What’s New in 2.1 }

* Problem: "{ ... }"
* Works Well: "{ ... }"

### What To Do
{ what designers, developers, others need to do }

### How it Helps
{ briefly explain the user experience – why this is needed }

{ how it makes content accessible for people with disabilities }

{ can include additional benefits beyond accessibility (e.g., using phone in bright sunlight) }

### [ More Details | Description ]
{ as succinct as feasible to get across important points. put supporting information elsewhere and can put links to it }

### [ Getting Started ]
{ not sure we’ll have this section. some COGA Design Patterns do and some do not. }

### Examples
{ Examples of barriers. Examples of solutions. }

### Support
{ how/which technologies support it or not - user agents such as browsers and assistive technology }

### Related WCAG
{ list guidelines, success criteria, techniques }

### Related Resources
{ only list here if particularly useful ; otherwise, delete this section }

### Key Terms
{ only list here if needed ; otherwise, delete this section ? can use WCAG termsor to add new one }

### Editorial Notes
These notes are not for the published document — just for us working through ideas.
{ notes here :- }

[View of what the initial state of the template looks like.](https://deploy-preview-21--wai-wcag-supplemental.netlify.app/lv/template/)

**Comment by shawna-slh** [comment]
Please note that the headings and content are based on what is currently in:
* Understanding WCAG
* WCAG Techniques
* COGA Supplemental Guidance -- that is, the Design Patterns from the Cognitive and Learning Disabilities Task Force)

The straw proposal for Low Vision Supplemental Guidance Template munges those.

Here is a **[table with the headings in the proposed template along with the headings in the COGA, Understanding, Techniques](https://w3c.github.io/low-vision-a11y-tf/sandbox/template_sections.html)**.

It seems good to use headings, order, and content that is mostly in sync with those existing resources, and not do something different unless there is a strong reason for it.

**Comment by shawna-slh** [comment]
In case there are several points to discuss, in order to help keep them straight, I created separate Discussions for these two points:
* [Template - User mini-personas or not? #142](https://github.com/w3c/low-vision-a11y-tf/discussions/142)
* [Template section heading - User Stories | User Needs | User Experience ... #140](https://github.com/w3c/low-vision-a11y-tf/discussions/140)

**Comment by mraccess77** [comment]
* Sam would like to have visual examples of what is gone right and wrong 
* Move examples earlier - could also move up with user stories
* Getting started moving up with more details

**Comment by mraccess77** [comment]
Bhoomika: need clarification on difference between getting started and "What to do".

**Comment by mraccess77** [comment]
Andy: tie examples with personas - link with mini personas - get audience to identify.  Thread personas through discussion.
